samples/embedded-auth-with-sdk/src/test/java/pages/Page.java

```diff
@@ -30,7 +30,7 @@
 
 public class Page {
 
-    private static final int RETRY_COUNT = 5;
+    private static final int RETRY_COUNT = 10;
```

Comment:
I think most other SDKs are retrying between 30-90 times. But I think by adding resend code functionality to our test logic, we would make these tests more reliable. See conversation here: https://okta.slack.com/archives/G01G626KYBA/p1623354657050900